src/widget/index.tsx

```diff
@@ -40,3 +40,7 @@ function injectChat() {
     );
 
 }
+
+declare global {
+    interface Window { attachEvent: Function, botmanWidget: Widget }
+}
```

Comment:
I wasn't able to do this globally :/ I'm using something similar in another component as a result